RadixWallet/Clients/AccountsClient/AccountsClient+Interface.swift

```diff
@@ -6,6 +6,8 @@ public struct AccountsClient: Sendable {
 	/// Accounts on current network (active gateway)
 	public var getAccountsOnCurrentNetwork: GetAccountsOnCurrentNetwork
 
+	public var getHiddenAccountsOnCurrentNetwork: GetHiddenAccountsOnCurrentNetwork
```

Comment:
Could extend `EntitiesVisibilityClient` to load all hidden entities.

Reply:
I feel `EntitiesVisibilityClient` is more about controlling visibilty, i.e. hiding/unhiding, whereas if I wanna query (hidden) accounts, it feels natural to use `accountsClient`.

Reply:
It also has querying though, and you could load all of hidden entities in one go, like `getHiddentEntities`. Still, no hard preference on my side here.